proto/account/service.proto

```diff
@@ -215,721 +213,42 @@ message GetAPIKeyBySearchingAllEnvironmentsResponse {
   EnvironmentAPIKey environment_api_key = 1;
 }
 
-option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
-  tags: [ { name: "Account" }, { name: "API Key" } ]
-  responses: {
-    key: "503"
-    value: {
-      description: "Returned for internal errors."
-      schema: { json_schema: { ref: ".google.rpc.Status" } }
-      examples: {
-        key: "application/json"
-        value: '{ "code": 13, "message": "internal", "details": [] }'
-      }
-    }
-  }
-};
+message CreateSearchFilterRequest {
+  string email = 1;
+  string organization_id = 2;
+  string environment_namespace = 3;
+  CreateSearchFilterCommand command = 4;
+}
+
+message CreateSearchFilterResponse {}
 
 service AccountService {
-  rpc GetMe(GetMeRequest) returns (GetMeResponse) {
```

Comment:
Why did you remove the API descriptions?

Reply:
I thought the code had changed when I ran the make command, so I committed it. I'll change it back.